### DISCUSSION TITLE
Using USB mass storage on Kobo Aura ONE

### DISCUSSION THREAD

**Discussion by DerpFromAbove** [koreader/koreader#8205] [Q&A] [answered]
Hello guys
When I recently started using KOReader on my Aura ONE, I noticed a feature in the menu called "USB mass storage". The user guide post does mention that you can use USB mass storage from within KOReader on Kobo devices. Now my question is, how exactly do I use it, what do I use it for and is it possible for me to plug in a USB stick loaded with PDFs and read them within KOReader without having to transfer them onto the e-reader first? 

Sadly, I currently do not possess a compatible USB stick to try this out with, so I am hoping some of you guys got experience with this and can help me out. Thanks in advance

Best regards

DerpFromAbove

**Comment by Frenzie** [comment, marked as answer]
No, it just means the device can present itself as USB storage to the computer.

**Reply by DerpFromAbove** [reply]
Well, shit
God damn Kobo and their insistence to use an 8gig micro SD in this day and age.
